src/extensions/service-vendors/bandwidth/messaging.js

```diff
@@ -178,19 +178,38 @@ export async function postMessageSend({
         ...changes
       }
     : {};
+  const organizationContact = {
+    contact_number: message.contact_number,
+    organization_id: organization.id,
+    service: "bandwidth"
+  };
   if (response && response.statusCode === 202 && response.result) {
     changesToSave.service_id = response.result.id;
+    organizationContact.status_code = 1;
+    organizationContact.user_number = response.result.from;
+    cacheableData.campaignContact.updateStatus(
```

Comment:
Must call `updateStatus` for the correct `campaign_contact` row to be linked to inbound messages when REDIS_CONTACT_CACHE=1 in the .env file